__tests__/app.test.js

```diff
@@ -109,6 +109,39 @@ describe("app", () => {
     });
   });
 
+  describe("GET /api/reviews/:review_id/comments", () => {
```

Comment:
Great testing so far, but also needs error handling to be tested:
A 404 for an id with no reviews
a 400 for an invalid id e.g /api/reviews/**egg**/comments
And finally, a 200 for a review with no comments, that should respond with an empty array of comments